src/main/java/org/cryptomator/common/vaults/Vault.java

```diff
@@ -339,6 +339,12 @@ public Path getPath() {
 		return vaultSettings.path().getValue();
 	}
 
+	public Path getCiphertextPath(Path cleartextPath) throws IOException {
```

Comment:
From my suggestion below: 
```java

	public Path getCiphertextPath(Path relativeCleartextPath) throws IOException {
		if(relativeCleartextPath.isAbsolute()) {
			throw new IllegalArgumentException("Input path must be relative, but is absolute: "+relativeCleartextPath);
		}
		var fs = cryptoFileSystem.get();
		var cryptoPath = fs.getRootDirectories().iterator().next().resolve(relativeCleartextPath.toString());
		return fs.getCiphertextPath(cryptoPath);
	}
```

Reply:
Fixed in https://github.com/cryptomator/cryptomator/pull/2592/commits/c97dbdd169f2de8baaed1dd3686494f69889d954